### PR TITLE
Module for random password generation

### PR DESCRIPTION
**Pull request by mkreisel**
Adds a module to (securely) randomly generate a password using alphanumeric characters. With this we should have the utilities we need for encryption/decryption endpoints that will be added in the next PR.

### CODE REVIEW

File: encryption/src/main/java/com/diyasylum/encryption/SecretBox.java [hunk 1]
```diff
@@ -30,6 +30,7 @@ public SecretBox(String password) {
 
   public Optional<String> decrypt(EncryptedMessage encryptedMessage) {
     try {
+      // TODO: Add input validation
```
Comment (Bachmann1234) [inline encryption/src/main/java/com/diyasylum/encryption/SecretBox.java]: I suggest filing this as an issue with a description and having this comment be a link to that

**Comment by Bachmann1234** [comment]
I suspect this module was not added to coverage reporting so I would not expect it to come back. I'll look into this soon. Maybe tonight.

**Comment by mkreisel** [comment]
@Bachmann1234 is there something I need to do to get it added to coverage?

**Comment by Bachmann1234** [comment]
@mkreisel actually looking at it you should not have to. Let me see if its something im missing

**Pull request merged**